CONTRIBUTING.md

```diff
@@ -1,55 +1,21 @@
-## Contributing to expressjs.com
+# Contributing to expressjs.com
 
-This repository is only for issues related to the website [http://expressjs.com](http://expressjs.com). For issues related to Express, the framework, go to [https://github.com/expressjs/express](https://github.com/expressjs/express).
+### The Offical Documentation of the Express Framework
 
-Feel free to make changes to the template files or the document files. The supporting docs are located in their respective directories, and the API docs are located under the `_includes` directory.
+Welcome to the repository for [https://expressjs.com](http://expressjs.com), the source for the official documentation of the Express framework.
 
-Content on this site is licensed under a Creative Commons Attribution-ShareAlike 3.0 United States License.  See https://creativecommons.org/licenses/by-sa/3.0/us/ for a layman's summary; 
-See [LICENSE.md](LICENSE.md) for the full license.
-
-## Contributing translations
-
-We highly encourage community translations! We no longer have professional translations, and we believe in the power of our community to provide accurate and helpful translations.
-
-The documentation is translated into these languages:
-- English (`en`)
-- Spanish (`es`)
-- French (`fr`)
-- Italian (`it`)
-- Indonesian (`id`)
-- Japanese (`ja`)
-- Korean (`ko`)
-- Brazilian Portuguese (`pt-br`)
-- Russian (`ru`)
-- Slovak (`sk`)
-- Thai (`th`)
-- Turkish (`tr`)
-- Ukrainian (`uk`)
-- Uzbek (`uz`)
-- Simplified Chinese (`zh-cn`)
-- Traditional Chinese (`zh-tw`)
 
-To find translations that need to be done, you can [filter for merged PRs](https://github.com/expressjs/expressjs.com/pulls?q=is%3Apr+is%3Aclosed+label%3Arequires-translation-es) that include the tag for your language, such as `requires-translation-es`.
+- *Please note: We are no longer accepting unsolicited pull requests*. Before any PRs will be merged they must first be submitted as issues, and approved. 
```

Comment:
hmmm.  when was this decided?

Reply:
Has it been discussed anywhere?

Reply:
It wasn't, put it the main point of this PR (for me). It was been noted in various pull requests before that were closed, since the work was not approved first, and the person's work was wasted.
#1608 had noted it, but no one really said yeh or neh. Can remove this, but does not solve that issue and may continue to happen.

Reply:
I did not remove this idea, but I've updated the language to be less assertive and more polite. Just in case it was the tone that was the issue and not the message itself. 

> _Please note: We are no longer accepting unsolicited pull requests._
> 
> - All this means is that we ask that you submit an issue describing your task first so it can get approved. After this you can submit a pull request.

Reply:
>  It was been noted in various pull requests before that were closed, since the work was not approved first, and the person's work was wasted.

Can you link me to some examples? While we do like when folks open issues before doing large amounts of work on a PR, we should never be "closing since the work was not approved first". That is VERY opposed to the ideals of the TC and if folks are doing that we need to make sure we clear up the goals and approaches we want to take with new contributors.

> Please note: We are no longer accepting unsolicited pull requests.

This language is very objectionable to me. Maybe we need to get the website team together on a TC meeting and have a coversation? It is nearly impossiable to keep up with all the conversations, but I am very concerned now that maybe the TC needs to jump into this topic.

Reply:
I am for sure not faulting you for writing it out, especially if that is your experience. And I would not ask for changes in here unless we were going to re-open the PR. My main point was that reading this experience it felt like there was a gap in how I (and based on conversations with others on the TC and project) feel things should be running. So as I was catching up on notifications I didn't want this to slip by unnoticed if we needed to address a situation that is counter to our governance and ideals.

There is alot on here, and as I have not been following the website updates closely I dont want to assume anything, but this is why I thought maybe we should all schedule time on the working call for the folks to get together and make sure we were all on the same page for how to run the project contributions.